pkg/controllers/controllers.go

```diff
@@ -66,5 +67,6 @@ func NewControllers(
 		metricsprovisioner.NewController(kubeClient),
 		counter.NewController(kubeClient, cluster),
 		deprovisioning.NewController(clock, kubeClient, provisioner, cloudProvider, eventRecorder, cluster),
+		inflightchecks.NewController(clock, kubeClient, eventRecorder, cloudProvider, cluster),
```

Comment:
Does it ever make sense to extend this beyond in flight?
```suggestion
		nodeanalysis.NewController(clock, kubeClient, eventRecorder, cloudProvider, cluster),
```

Reply:
Maybe? I'm not fixed on the name, could possibly be incorporated into a future node health check.  These are currently weird things where we don't know what to do and operators need to intervene which may not make sense for any sort of automated health code.